internal/worker_versioning_info.go

```diff
@@ -237,6 +237,15 @@ func taskQueueVersionInfoFromResponse(response *taskqueuepb.TaskQueueVersionInfo
 	}
 }
 
+func detectEnhancedNotSupported(response *workflowservice.DescribeTaskQueueResponse) error {
+	// A server before 1.24 returns a non-enhanced proto, which only fills `pollers` and `taskQueueStatus` fields
```

Comment:
Can probably just check task queue status. Pollers seems unnecessary and may be empty in both cases.

Reply:
I don't know exactly what the old implementation did, I'm just being overly cautious here...

Reply:
I think you can ignore poller count, that can be empty in both cases